### PR TITLE
Issue/#23

### PR DESCRIPTION
**Pull request by HannesSundin**
#23 and or #21

### CODE REVIEW

File: pages/members.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: `${user.pnr}+${user.password}`,
+            }
+        })
+            .then(res => res.json())
+            .then(data => setMembers(data))
+            .catch((err) => console.log(err.statusText));
+    }, [])
+    return <MembersList members={members}/>
```
Comment (StelFoog) [inline pages/members.js:35]: Should be in a `<main>` tag

**Review by StelFoog — CHANGES_REQUESTED** [review]

File: pages/members.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        </div>)
+}
+
+export default function Members() {
+    const [members, setMembers] = useState([])
+    const user = useSelector(selectUser)
+    useEffect(() => {
+        const baseURL = (process.env.NODE_ENV === "development" ?
+            "http://localhost:3000/" : "https://speakeasy-ten.vercel.app/")
+        fetch(baseURL + "api/members/get", {
```
Comment (StelFoog) [inline pages/members.js:24]: fetch automatically adds the url it is on when url doesn't start with `http` or `https`, use `fetch('/api/members/get')` instead

**Review by StelFoog — CHANGES_REQUESTED** [review]

File: pages/members.js (hunk 2 above)
Comment (HannesSundin) [reply pages/members.js:24]: i had an error with something about absolute url, but youre right it works now

**Review by StelFoog — APPROVED** [review]
lgtm

**Pull request merged**